### PR TITLE
📚Add Introduction and Update Structure

### PR DESCRIPTION
**Pull request by 5ebastianMeier**
## What did you change?

I've added content to the introduction sections and tried to add links wherever appropriate.

The main goal of this PR is to give someone who visits the documentation the first time a better overview of its contents and a brief understanding of the purpose of each part of the documentation.

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: getting-started/installation.md [hunk 1]
```diff
@@ -7,10 +7,20 @@ kann man
...
-und nutzen. Das Ergebnis sieht so aus:
+Wurde die Anwendung erfolgreich installiert und ausgeführt, öffnet sich die [Prozessübersicht](./../bpmn-studio/beschreibung-ansichten.md#process-definition-list).
 
 ![BPMN-Studio](images/bpmn-studio.png)
+
+### Mac
+
+Die oben verlinkten Releases enthalten eine `.dmg`-Datei.
+
+Diese Datei kann direkt ausgeführt werden oder zur Installation in den `Programme`-Ordner verschoben werden.
```
Comment (ElRaptorus) [inline getting-started/installation.md]: Typo: `...ausgeführt werden, oder...`

Comment (cmg-dev) [reply getting-started/installation.md]: na, geht, kann man auch ohne schreiben.

File: bpmn-studio/beschreibung-ansichten.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Beschreibung der Dialoge
 
-## Process Definition List
+## Process Definition List {#process-definition-list}
 
 ![Process Definition List](./process-definition-list.png)
```
[the remark this answers is not in the document]
Comment (cmg-dev) [reply bpmn-studio/beschreibung-ansichten.md]: bitte issue dazu erstellen. Im anderen PR lösen.

**Comment by cmg-dev** [comment]
Generell bitte auf 80 Zeichen/Zeile achten.

File: bpmn-studio/beschreibung-ansichten.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ElRaptorus) [reply bpmn-studio/beschreibung-ansichten.md]: Da gibts tatsächlich schon eins zu. Hat bisher nur scheinbar keiner bemerkt, weil keiner als Verantwortliche eingetragen war.

https://github.com/process-engine/documentation/issues/88

File: getting-started/installation.md (hunk 1 above)
Comment (ElRaptorus) [reply getting-started/installation.md]: Na gut. 🙃

**Pull request merged**